neo/SmartContract/Contract.py

```diff
@@ -91,8 +91,8 @@ def Create(redeemScript, parameterList, publicKeyHash):
     @staticmethod
     def CreateMultiSigRedeemScript(m, publicKeys):
 
-        if m < 2 or m > len(publicKeys) or len(publicKeys) > 1024:
-            raise Exception('Invalid keys')
+        if m < 1 or m > len(publicKeys) or len(publicKeys) > 1024:
```

Comment:
The [C# code](https://github.com/neo-project/neo/blob/879952a8259347db3f04211502efebe1dec202cd/neo/SmartContract/Contract.cs#L75) also allows 1 signature,  now this does so as well.